frontend/src/js/services/services.js

```diff
@@ -104,6 +104,53 @@
             angular.element("#sim-loader").fadeOut();
             angular.element(".web-container").removeClass('low-screen');
         };
+
+        this.check = function(password){
+
+           //Regular Expressions.  
+            var regex = new Array();
+            regex.push("[A-Z]"); //Uppercase Alphabet.  
```

Comment:
There is no need to write 4 statements to do similar operation. You can push multiple items in a array in single statement i.e. `array.push(item1, item2, item3 ......)`